src/detector/detector_app.py

```diff
@@ -24,25 +24,34 @@
 import logging
 import os
 
+from src.detector.implementations import abstract_marker_interface, aruco_marker_implementation
+
 
 logger = logging.getLogger(__name__)
 
 
 def create_app() -> FastAPI:
     detector_configuration_filepath: str = os.path.join(os.path.dirname(__file__), "..", "..", "data", "config.json")
     detector_configuration: DetectorConfiguration
+    #camera_interface: AbstractCameraInterface
+    marker_interface: abstract_marker_interface
+
     with open(detector_configuration_filepath, 'r') as infile:
         detector_configuration_file_contents: str = infile.read()
         detector_configuration_dict = hjson.loads(detector_configuration_file_contents)
         detector_configuration = DetectorConfiguration(**detector_configuration_dict)
-    detector = Detector(detector_configuration=detector_configuration)
+
+    #camera_interface = Detector.USBWebcamWithOpenCV()
+    marker_interface = aruco_marker_implementation.ArucoMarker()
+    
+    detector = Detector(detector_configuration=detector_configuration,marker_interface=marker_interface)
     detector_app = FastAPI()
 
     # CORS Middleware
     origins = ["http://localhost"]
     detector_app.add_middleware(
         CORSMiddleware,
-        allow_origins=origins,
```

Comment:
I think this was the change I made during troubleshooting. It's probably best to revert this to `origins`.